### PR TITLE
shorah version bump to 1.1.1

### PR DESCRIPTION
**Pull request by DrYak**
New version of Shorah added to the existing ones.
(Also includes a patch to make the C/C++ command line utilities return nicely with `-h/`--help` options)

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by armintoepfer** [comment]
Rather ping @ozagordi, who is the owner

File: recipes/shorah/1.1.1/shorah_nice_exit.patch [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
++    exit(status);
+ }
+ 
+ int main(int argc, char** argv)
+@@ -439,7 +439,7 @@ int main(int argc, char** argv)
+                 help();
+                 break;
+             case 'h':
+-                help();
++                help(0);
```
Comment (SoapZA) [inline recipes/shorah/1.1.1/shorah_nice_exit.patch]: @DrYak please change all call sites to use `EXIT_SUCCESS` and `EXIT_FAILURE` macros. Then, don't include the patch, and instead send in a PR to our shorah upstream. We should avoid carrying patches upstream if at all possible (which it is, seeing that we are upstream).

Once the patch is integrated, I will cut a 1.1.2 release instead.

Comment (DrYak) [reply recipes/shorah/1.1.1/shorah_nice_exit.patch]: of course the end plan is to integrate it upstream.  (pull request will be underway later this, once I find some free time during the Bern NGS Meeting).


But @sposadac needs it by tomorrow, so this 1.1.1 pull request is the temporary emergency solution, until 1.1.2.

**Comment by armintoepfer** [comment]
Duct tape programming 101

**Comment by DrYak** [comment]
Duck Tape

Yup...

I'll be starting on 1.1.2 shortly...

**Comment by DrYak** [comment]
Trying to [remove my layer of duck tape](https://github.com/cbg-ethz/shorah/pull/37) without loosing too many feathers...

**Comment by DrYak** [comment]
...and [as promised](https://github.com/bioconda/bioconda-recipes/pull/6824).

**Pull request merged**